<html>
<head>
<title>TestNG:  Test</title>
<link href="../testng.css" rel="stylesheet" type="text/css" />
<link href="../my-testng.css" rel="stylesheet" type="text/css" />

<style type="text/css">
.log { display: none;} 
.stack-trace { display: none;} 
</style>
<script type="text/javascript">
<!--
function flip(e) {
  current = e.style.display;
  if (current == 'block') {
    e.style.display = 'none';
    return 0;
  }
  else {
    e.style.display = 'block';
    return 1;
  }
}

function toggleBox(szDivId, elem, msg1, msg2)
{
  var res = -1;  if (document.getElementById) {
    res = flip(document.getElementById(szDivId));
  }
  else if (document.all) {
    // this is the way old msie versions work
    res = flip(document.all[szDivId]);
  }
  if(elem) {
    if(res == 0) elem.innerHTML = msg1; else elem.innerHTML = msg2;
  }

}

function toggleAllBoxes() {
  if (document.getElementsByTagName) {
    d = document.getElementsByTagName('div');
    for (i = 0; i < d.length; i++) {
      if (d[i].className == 'log') {
        flip(d[i]);
      }
    }
  }
}

// -->
</script>

</head>
<body>
<h2 align='center'>Test</h2><table border='1' align="center">
<tr>
<td>Tests passed/Failed/Skipped:</td><td>3/2/0</td>
</tr><tr>
<td>Started on:</td><td>Mon Aug 10 19:55:44 IST 2020</td>
</tr>
<tr><td>Total time:</td><td>94 seconds (94876 ms)</td>
</tr><tr>
<td>Included groups:</td><td></td>
</tr><tr>
<td>Excluded groups:</td><td></td>
</tr>
</table><p/>
<small><i>(Hover the method name to see the test class name)</i></small><p/>
<table width='100%' border='1' class='invocation-failed'>
<tr><td colspan='4' align='center'><b>FAILED TESTS</b></td></tr>
<tr><td><b>Test method</b></td>
<td width="30%"><b>Exception</b></td>
<td width="10%"><b>Time (seconds)</b></td>
<td><b>Instance</b></td>
</tr>
<tr>
<td title='annotationPart1.TestNGDemo3.Browser()'><b>Browser</b><br>Test class: annotationPart1.TestNGDemo3<br>Parameters: 9740673180, raghubn</td>
<td><div><pre>java.lang.AssertionError: User Not able to login Sucessfully - Invalid Credentails expected [true] but found [false]
	at annotationPart1.TestNGDemo3.Browser(TestNGDemo3.java:32)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 13 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace933172290", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace933172290'><pre>java.lang.AssertionError: User Not able to login Sucessfully - Invalid Credentails expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at annotationPart1.TestNGDemo3.Browser(TestNGDemo3.java:32)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
</pre></div></td>
<td>20</td>
<td>annotationPart1.TestNGDemo3@67117f44</td></tr>
<tr>
<td title='annotationPart1.TestNGDemo3.Browser()'><b>Browser</b><br>Test class: annotationPart1.TestNGDemo3<br>Parameters: raghubn2, raghubn</td>
<td><div><pre>java.lang.AssertionError: User Not able to login Sucessfully - Invalid Credentails expected [true] but found [false]
	at annotationPart1.TestNGDemo3.Browser(TestNGDemo3.java:32)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 13 stack frames</pre></div><a href='#' onClick='toggleBox("stack-trace533631442", this, "Click to show all stack frames", "Click to hide stack frames")'>Click to show all stack frames</a>
<div class='stack-trace' id='stack-trace533631442'><pre>java.lang.AssertionError: User Not able to login Sucessfully - Invalid Credentails expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at annotationPart1.TestNGDemo3.Browser(TestNGDemo3.java:32)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
</pre></div></td>
<td>14</td>
<td>annotationPart1.TestNGDemo3@67117f44</td></tr>
</table><p>
<table width='100%' border='1' class='invocation-passed'>
<tr><td colspan='4' align='center'><b>PASSED TESTS</b></td></tr>
<tr><td><b>Test method</b></td>
<td width="30%"><b>Exception</b></td>
<td width="10%"><b>Time (seconds)</b></td>
<td><b>Instance</b></td>
</tr>
<tr>
<td title='annotationPart1.TestNGDemo3.Browser()'><b>Browser</b><br>Test class: annotationPart1.TestNGDemo3<br>Parameters: 9740673180, raghubn@123</td>
<td></td>
<td>44</td>
<td>annotationPart1.TestNGDemo3@67117f44</td></tr>
<tr>
<td title='annotationPart1.TestNGDemo2.Cat1()'><b>Cat1</b><br>Test class: annotationPart1.TestNGDemo2</td>
<td></td>
<td>22</td>
<td>annotationPart1.TestNGDemo2@6f79caec</td></tr>
<tr>
<td title='annotationPart1.TestNGDemo1.VarifyPageTitle()'><b>VarifyPageTitle</b><br>Test class: annotationPart1.TestNGDemo1</td>
<td></td>
<td>14</td>
<td>annotationPart1.TestNGDemo1@1ef7fe8e</td></tr>
</table><p>
</body>
</html>